### DISCUSSION TITLE
Symfony Config in Monolog.php & custom file

### DISCUSSION THREAD

**Discussion by abatalib** [symfony/symfony#52662] [Q&A]
Hi,
I would like to create a monolog custom log file in an independent file (not dev.log) by using the monolog.php not yaml, there are a symfony doc for that, but doesn't work, here is.the doc https://symfony.com/doc/current/logging/channels_handlers.html

Thanks

**Comment by ghost** [comment]
Saying **"doesn't work"** doesn't mean anything. When your car is broken do you call the service station and say **"Hello. My car is broken. Thanks"**?
Please add more info like error logs, code samples etc.

**Comment by abatalib** [comment]
Assuming i want to create a perso.log file instead of the default dev.log file, i put the instructions as mentionned by symfony doc (in monolog.php not yaml), but the log informations is displayed in dev.log not in perso.log.

**Reply by stof** [reply]
Please provide a reproducing case (and also check whether you actually log things in the channel for which you used a custom file and not a different channel)

**Reply by ghost** [reply]
As I said before, please provide at least code samples like monolog.php content, excerpt from the class where logger instance is logging incorrectly, excerpt from services.yaml where you configure the injected logger (if you use autowiring then please say it)
